scrapy/_monkeypatches.py

```diff
@@ -1,21 +1,4 @@
-import six
 from six.moves import copyreg
-
-
-if six.PY2:
-    from urlparse import urlparse
-
-    # workaround for https://bugs.python.org/issue7904 - Python < 2.7
-    if urlparse('s3://bucket/key').netloc != 'bucket':
-        from urlparse import uses_netloc
-        uses_netloc.append('s3')
-
-    # workaround for https://bugs.python.org/issue9374 - Python < 2.7.4
```

Comment:
I think we shouldn't remove the second workaround, as it may be needed in 2.7 < Python < 2.7.4

Reply:
Yep my bad, python 2.7.4 is still permitted . I am newbie to the whole project so this may sound stupid but why is the Travis CI build is failing? I noticed it with other pull requests too.

Reply:
Added the second workaround in  the new commit.

Reply:
> why is the Travis CI build is failing? I noticed it with other pull requests too.

At the moment #3913 and #3917. If you stick around, though, I’m sure you’ll have the opportunity to see it start failing for other reasons; it happens now and then :slightly_smiling_face: